src/templates/show.html

```diff
@@ -29,132 +29,141 @@ <h1 id='pageTitle' {% if hideTitle %}hidden{% endif %}>{{ App.pageTitle }}</h1>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'status' and App.Request.query.get('sort') == 'desc') ? 'active' }}' data-query='status-desc' href='#'>{{ 'Status'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'status' and App.Request.query.get('sort') == 'asc') ? 'active' }}' data-query='status-asc' href='#'>{{ 'Status (inverted)'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'id' and App.Request.query.get('sort') == 'desc') ? 'active' }}' data-query='id-desc' href='#'>{{ 'Highest ID'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'id' and App.Request.query.get('sort') == 'asc') ? 'active' }}' data-query='id-asc' href='#'>{{ 'Lowest ID'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'customid' and App.Request.query.get('sort') == 'desc') ? 'active' }}' data-query='customid-desc' href='#'>{{ 'Highest Custom ID'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'customid' and App.Request.query.get('sort') == 'asc') ? 'active' }}' data-query='customid-asc' href='#'>{{ 'Lowest Custom ID'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'comment' and App.Request.query.get('sort') == 'desc') ? 'active' }}' data-query='comment-desc' href='#'>{{ 'Recently commented'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'title' and App.Request.query.get('sort') == 'asc') ? 'active' }}' data-query='title-asc' href='#'>{{ 'Title (alphabetical)'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'title' and App.Request.query.get('sort') == 'desc') ? 'active' }}' data-query='title-desc' href='#'>{{ 'Title (inverted)'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'user' and App.Request.query.get('sort') == 'asc') ? 'active' }}' data-query='user-asc' href='#'>{{ 'User (alphabetical)'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'user' and App.Request.query.get('sort') == 'desc') ? 'active' }}' data-query='user-desc' href='#'>{{ 'User (inverted)'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'rating' and App.Request.query.get('sort') == 'desc') ? 'active' }}' data-query='rating-desc' href='#'>{{ 'Highest rating'|trans }}</a>
-          <a class='dropdown-item {{ (App.Request.query.get('order') == 'rating' and App.Request.query.get('sort') == 'asc') ? 'active' }}' data-query='rating-asc' href='#'>{{ 'Lowest rating'|trans }}</a>
+          <button type='button' class='btn btn-dropdown-item dropdown-item {{- (App.Request.query.get('order') == 'date' and App.Request.query.get('sort') == 'desc') ? ' active' }}' data-query='date-desc'>{{ 'Newest'|trans }}</a>
```

Comment:
on this line, the `a` wasn't replaced. Shouldn't this be caught by the validation process?

Reply:
Yes, that should be detected. For some reason the html validate is not working so smooth. Maybe because the elements are hidden.

Reply:
This might be due to the fact that we test the DOM and not the raw page source. So the electron browser might swallow that and converts it into a valid `<button>...</button>`